### ISSUE TITLE
template error on Django 3.0.x

### ISSUE THREAD

**Issue by KushGoyal** [django-otp/django-otp#15]
I am using `admin.site.__class__ = OTPAdminSite` to set the default admin site class.

Below is the error when using with Django 3.0.x:

> 'admin_static' is not a registered tag library. Must be one of: admin_list admin_modify admin_urls cache i18n l10n log static tz

```html

{% extends "admin/base_site.html" %}
{% load i18n admin_static %} 
{% block extrastyle %}
{{ block.super }}
<link rel="stylesheet" type="text/css" href="{% static "admin/css/login.css" %}" />
{{ form.media }}
 
<style type="text/css">
input#id_otp_token,
select#id_otp_device
{


```

**Comment by moggers87** [comment]
Fixed in #14

**Comment by psagers** [comment]
Unfortunately, this may be more complicated to fix than it appears, due to #10. Note that you can always work around this by making a copy of the template and overriding `OTPAdminSite.login_template` yourself.

**Comment by psagers** [comment]
Fixed in 0.7.5 (cf15b670c732d9d37f57a7504003712392ab0134). If #10 recurs, more cleverness may be needed.
